source/support/rotate-secrets/index.html.md.erb

```diff
@@ -0,0 +1,186 @@
+
+[Grafana]: https://grafana-paas.cloudapps.digital/org/apikeys
+
+#### `paas-username` and `paas-password`
+
+Change the password by [triggering a password reset][], which gets sent to the govuk-accounts-developers Google Group.
+
+[triggering a password reset]: https://admin.london.cloud.service.gov.uk/password/request-reset
+
+#### `sentry-dsn`
+
+Generate a new DSN from the Sentry project settings page, which you
+can reach by:
+
+1. Navigating to the [GOV.UK Sentry organisation][].
+2. Selecting the project from the drop-down list.
+3. Clicking the cog icon next to the project name.
+
+[GOV.UK Sentry organisation]: https://sentry.io/organizations/govuk
+
+#### `slack_webhook_url`
+
+Tell IT Support, through the [Web Helpdesk][], that our Slack webhook
+has been leaked, and that we need a new one.
+
+[Web Helpdesk]: https://gdshelpdesk.digital.cabinet-office.gov.uk
+
+### Secrets used by the account manager
+
+#### `basic-auth-username` and `basic-auth-password`
+
+These are arbitrary strings. After changing them, update the
+"deployed applications" Trello card.
+
+#### `bigquery-credentials-production`
+
+Generate new JSON credentials in the [Google Cloud Console][], and
+deactivate the old credentials.
+
+[Google Cloud Console]: https://console.cloud.google.com/iam-admin/serviceaccounts?orgonly=true&project=govuk-account-analysis-db&supportedpurview=project
+
+#### `notify-api-key-production` and `notify-api-key-staging`
+
+Generate a new API key from [GOV.UK Notify][], and deactivate the old
+API key.
+
+[GOV.UK Notify]: https://www.notifications.service.gov.uk/
+
+#### `oidc-pepper-production` and `oidc-pepper-staging`
+
+Generate an arbitrary string using `rake secret`.
+
+This secret is used to [generate subject identifiers][], and so
+rotating it will cause services to see a new subject identifier for a
+returning user. This means that services will not be able to identify
+returning users, and will treat them as new users.
+
+If this secret needs to be rotated, we will need to engage with other
+services, so that they can migrate their data.
+
+[generate subject identifiers]: https://github.com/alphagov/govuk-account-manager-prototype/blob/7e69b8cb8daec1e6e23d32374df3f8a1fa45b477/config/initializers/doorkeeper_openid_connect.rb#L34-L36
+
+#### `oidc-signing-key-production` and `oidc-signing-key-staging`
+
+Generate a new RSA private key.
+
+#### `password-pepper-production` and `password-pepper-staging`
+
+Generate an arbitrary string using `rake secret`.
+
+This secret is used to [hash account passwords][], and so rotating it
+will invalidate all of the current passwords. This means that users
+will have to reset their password to log in.
+
+If this secret needs to be rotated on short notice, we may want to
+send an email to all users explaining the problem. If it can be
+rotated over a longer period, we could rehash passwords when users log
+in, and only disable the old pepper when most users have been changed.
```

Comment:
@barrucadu Should "the old pepper" say something else?

Reply:
Changed it to "the old value"